python/ray/autoscaler/kubernetes/example-ingress.yaml

```diff
@@ -108,178 +108,181 @@ provider:
                   port: 8000
                   targetPort: 8000
 
-# Kubernetes pod config for the head node pod.
-head_node:
-    pod:
-        apiVersion: v1
-        kind: Pod
-        metadata:
-            # Automatically generates a name for the pod with this prefix.
-            generateName: ray-head-
 
-            # Must match the head node service selector above if a head node
-            # service is required.
-            labels:
-                component: ray-head
-        spec:
-            # Change this if you altered the autoscaler_service_account above
-            # or want to provide your own.
-            serviceAccountName: autoscaler
-
-            # Restarting the head node automatically is not currently supported.
-            # If the head node goes down, `ray up` must be run again.
-            restartPolicy: Never
-
-            # This volume allocates shared memory for Ray to use for its plasma
-            # object store. If you do not provide this, Ray will fall back to
-            # /tmp which cause slowdowns if is not a shared memory volume.
-            volumes:
-            - name: dshm
-              emptyDir:
-                  medium: Memory
-
-            containers:
-            - name: ray-node
-              imagePullPolicy: IfNotPresent
-              # You are free (and encouraged) to use your own container image,
-              # but it should have the following installed:
-              #   - rsync (used for `ray rsync` commands and file mounts)
-              #   - screen (used for `ray attach`)
-              #   - kubectl (used by the autoscaler to manage worker pods)
-              image: rayproject/ray:nightly
-              # Do not change this command - it keeps the pod alive until it is
-              # explicitly killed.
-              command: ["/bin/bash", "-c", "--"]
-              args: ["trap : TERM INT; sleep infinity & wait;"]
-              ports:
-                  - containerPort: 6379 # Redis port for Ray <= 1.10.0. GCS server port for Ray >= 1.11.0
-                  - containerPort: 22345 # Ray internal communication.
-                  - containerPort: 22346 # Ray internal communication.
+available_node_types:
```

Comment:
Would you mind deleting all of the configs in the kubernetes folder except for defaults.yaml?
(Optional, I can do it myself in a separate PR if needed.)